review-tools/addrev

```diff
@@ -16,7 +16,7 @@ my $my_email;
 foreach (@ARGV) {
     if (/^\@.+$/) {
         $args .= "--reviewer=$_ ";
-    } elsif (/^[-\w]+$/) {
+    } elsif (/^\w[-\w]+$/) {
```

Comment:
Side note: one character reviewer names are no longer allowed :wink:

Reply:
> Side note: one character reviewer names are no longer allowed :wink:

o :wink:

So it would have been better to use `/^\w[-\w]*$/`

Reply:
I'm going to fix this very minor issue if /when doing any further improvements of `addrev`.